components/atom/imagePanel/src/index.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+
+const BASE_16 = 16
+const HORIZONTAL_ALIGNMENTS = {
+  LEFT: 'left',
+  CENTER: 'center',
+  RIGHT: 'right'
+}
+
+const VERTICAL_ALIGNMENTS = {
+  TOP: 'top',
+  CENTER: 'center',
+  BOTTOM: 'bottom'
+}
+
+const TYPES = {
+  RESIZED: 'resized'
+}
+
+const hexToRgb = function (hex) {
+  const shorthandRegex = /^#?([a-f\d])([a-f\d])([a-f\d])$/i
+  hex = hex.replace(shorthandRegex, (m, r, g, b) => r + r + g + g + b + b)
+
+  var result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex)
+  return result ? {
+    r: parseInt(result[1], BASE_16),
+    g: parseInt(result[2], BASE_16),
+    b: parseInt(result[3], BASE_16)
+  } : {}
+}
+
+const getClassNames = function ({verticalAlign, horizontalAlign, classNames, resized}) {
+  return classnames(
+    'sui-AtomImagePanel',
+    `sui-AtomImagePanel--v-${verticalAlign}`,
+    `sui-AtomImagePanel--h-${horizontalAlign}`,
+    resized && `sui-AtomImagePanel-resized`,
+    classNames
+  )
+}
+
+const getStyles = function ({placeholderColor, overlayColor, src, overlayAlpha}) {
+  const url = `url(${src})`
+  if (overlayColor) {
+    const overlayColorRgb = hexToRgb(overlayColor)
+    const rgba = `rgba(${overlayColorRgb.r}, ${overlayColorRgb.g}, ${overlayColorRgb.b}, ${overlayAlpha})`
+    var gradient = `linear-gradient(${rgba}, ${rgba})`
+  }
+  return {
+    backgroundImage: gradient ? `${gradient}, ${url}` : url,
+    backgroundColor: placeholderColor
+  }
+}
+
+const AtomImagePanel = function ({src, children, ...props}) {
+  return (
+    <div className={getClassNames(props)} style={getStyles({src, ...props})}>
+      {children}
+    </div>
+  )
+}
+
+AtomImagePanel.displayName = 'AtomImagePanel'
+
+AtomImagePanel.propTypes = {
+  className: PropTypes.string,
+  children: PropTypes.node.isRequired,
+  /**
+   * Background image
+   */
+  src: PropTypes.string.isRequired,
+  /**
+   * Background color
+   */
+  placeholderColor: PropTypes.string.isRequired,
+  /**
+   * Gradient color
+   */
+  overlayColor: PropTypes.string,
+  /**
+   * Gradient opacity
+   * @type float between 0 and 1
+   */
+  overlayAlpha: PropTypes.number,
```

Comment:
According to zepplin, alpha is always `0.5`.
We should only make customizable what's meant to be by design.

Reply:
But acording to the defintion: 
It’s optional and has a certain amount of opacity that can be configured.

Reply:
Ok. I thought they meant the opacity could be configure by theme, like the color.